examples/capi-quickstart/capd-cluster.yaml

```diff
@@ -18,26 +17,14 @@ spec:
       cidrBlocks:
         - 10.128.0.0/12
   topology:
-    class: quick-start
+    class: capd-quick-start
     controlPlane:
       metadata: {}
       replicas: 1
     variables:
-      - name: imageRepository
-        value: ""
-      - name: etcdImageTag
-        value: ""
-      - name: coreDNSImageTag
-        value: ""
-      - name: podSecurityStandard
-        value:
-          audit: restricted
-          enabled: true
-          enforce: baseline
-          warn: restricted
```

Comment:
The other variables are optional, but this one is still needed for Calico to come up right?

Reply:
:thinking: It didn't seem to be. I tested this locally and it worked. Would have to think about why though to properly understand.

Reply:
Same it worked for me too 🤔